Scenarios/Networking/AVS-to-VNet-NewVNet/Bicep/Modules/AVSAuthorization.bicep

```diff
@@ -4,6 +4,9 @@ param PrivateCloudName string
 @description('The authorization key name to be created')
 param AuthKeyName string
 
+@description('Opt-out of deployment telemetry')
```

Comment:
You have only exposed this in the module, not the parent template, it will need to be passed in from the parent template otherwise users can't set this to true.
Recommendation:
 - Any module param shouldn't have a default value unless there is an explicit reason
 - TelemetryOptOut should always be exposed as a param in the scenario parent template with a default value of false.